docs/migration-to-NRI.md

```diff
@@ -0,0 +1,182 @@
+# Prerequisities
+- Up and running CRI Resource Manager
+- One of the two supported policies in use: balloons or topology-aware.
+- For other policies a little bit more work is required and the policies need to be 'ported'. This can be done by just following the example of how balloons or topology-aware was converted.
```

Comment:
```suggestion
- For other policies a little bit more work is required and the policies need to be 'ported'. This can be done by just following the example of how balloons or topology-aware were converted.
```